stytch/consumer/otp/email/types.go

```diff
@@ -0,0 +1,135 @@
+package email
+
+// !!!
+// WARNING: This file is autogenerated
+// Only modify code within MANUAL() sections
+// or your changes may be overwritten later!
+// !!!
+
+import (
+	"github.com/stytchauth/stytch-go/v8/stytch/consumer/attribute"
+)
+
+// LoginOrCreateParams: Request type for `Email.LoginOrCreate`.
+type LoginOrCreateParams struct {
+	// Email: The email address of the user to send the one-time passcode to. You may use sandbox@stytch.com to
+	// test this endpoint, see [Testing](https://stytch.com/docs/home#resources_testing) for more detail.
+	Email string `json:"email,omitempty"`
+	// ExpirationMinutes: Set the expiration for the Magic Link `token` in minutes. By default, it expires in 1
```

Comment:
Another instance of not getting the variation from the variation from the param text
```
export const expiration_minutes: ParamWithVariants = {
  description: `Set the expiration for the Magic Link \`token\` in minutes. By default, it expires in 1 hour. The minimum expiration is 5 minutes and the maximum is 7 days (10080 mins).`,
  name: 'expiration_minutes',
  type: 'int',
  variants: {
    otp: {
      description:
        'Set the expiration for the one-time passcode, in minutes. The minimum expiration is 1 minute and the maximum is 10 minutes. The default expiration is 2 minutes.',
    },